flink-streaming-connectors/flink-connector-rabbitmq/src/main/java/org/apache/flink/streaming/connectors/rabbitmq/RMQSink.java

```diff
@@ -76,6 +76,9 @@ public void open(Configuration config) throws Exception {
 		try {
 			connection = factory.newConnection();
 			channel = connection.createChannel();
+			if (channel == null) {
```

Comment:
Do you know under which circumstances this can occur and/or what a user can do to resolve it?